WordPress/twentytwentyone
# Reset the cover block overlay color to black and text to white

Fixes #389.

## Summary
In the editor and in frontend, resetted the overlay of cover block to black and the text inside to white.

## Relevant technical choices:
No.

## Test instructions
This PR can be tested by following these steps:

Add a cover block and add a background image
Add some texts inside, paragraph and heading
In the editor and in frontend, the cover block overlay should be black and the text inside white.


## Quality assurance
